operator/main.py

```diff
@@ -974,8 +997,8 @@ def deploy_storage_class():
             logging.info(logf("StorageClass already present, continuing with Apply",
                               manifest=filename))
 
-        # Deploy Storage Class
-        template(filename, namespace=NAMESPACE, kadalu_version=VERSION)
+        template(filename, namespace=NAMESPACE, kadalu_version=VERSION,
```

Comment:
Remove all other storage class files except the last one

```
storageclass-kadalu.disperse.yaml.j2
storageclass-kadalu.replica1.yaml.j2
storageclass-kadalu.replica2.yaml.j2
storageclass-kadalu.replica3.yaml.j2

storageclass-kadalu.yaml.j2
```

Reply:
Wont we cause regression for existing users if we remove it? Or because they have already set it up like this, no need for these in future versions?

Reply:
Already deployed SCs will not be touched, no new SCs for these types are not created.